Slambook/index.js

```diff
@@ -1,14 +1,26 @@
 let form;
 
 let allSlamArray=[];
+let formValue = {};
+let err ;
 //Write code below to select the form element
 form = document.querySelector('form')
```

Comment:
There is a syntax error in the 'forEach' callback function. It should not have any parameter passed to it.